tests/codex/sales/states/testpreparing.nim

```diff
@@ -27,3 +73,28 @@ suite "sales state 'preparing'":
   test "switches to filled state when slot is filled":
     let next = state.onSlotFilled(request.id, slotIndex)
     check !next of SaleFilled
+
+  proc createAvailability() =
+    let a = waitFor reservations.createAvailability(
```

Comment:
Why not `await` instead of `waitFor`?

Reply:
🤷‍♂️ I "stole" this originally from @benbierens work 😅 @benbierens why?

Reply:
I did that?... I don't know. I'm sure I stole it from somewhere as well.

Reply:
The "copy&paste" mafia 😂